package.json

```diff
@@ -11,13 +11,14 @@
     "start": "babel-node -x \".js\" -r dotenv/config src/server/index.js",
     "server-dev": "babel-node -x \".js\" src/server/index.js",
     "test": "mocha test/**/*.test.js --require ./test/dom.js --require esm --require ignore-styles --require @babel/register ./test --exit || true",
+    "create-postgres-database": "psql -U TO_DO -d postgres -a -f ./src/gateways/schema/schema.sql",
     "seed-postgres": "psql -U TO_DO -d inventory -a -f ./seeder/postgres_seeder/seed.sql",
+    "seed-mongo": "mongoimport --type csv -d inventory -c Products --headerline --drop ./seeder/seed_data/mockData.csv",
     "generate-data": "babel-node -x \".js\" ./seeder/postgres_seeder/data_generator/index.js",
     "react-dev": "webpack -d --watch",
     "build": "webpack --mode production"
   },
   "dependencies": {
-    "@babel/plugin-transform-runtime": "^7.12.1",
```

Comment:
**Annotation:** Babel's transform runtime plugin should fall under development dependencies.